src/risedevtool/src/task/meta_node_service.rs

```diff
@@ -174,8 +174,6 @@ impl Task for MetaNodeService {
         let mut cmd = self.meta_node()?;
 
         cmd.env("RUST_BACKTRACE", "1");
-        // FIXME: Otherwise, CI will throw log size too large error
-        // cmd.env("RW_QUERY_LOG_PATH", DEFAULT_QUERY_LOG_PATH);
```

Comment:
Is this still true?

Reply:
<img width="676" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/37948597/0d2f5ff2-8fbd-4e0f-aa8f-d5e7f63900b3">